modules/electromagnetics/test/tests/kernels/coupled_electrostatics/coupled_electrostatics.i

```diff
@@ -0,0 +1,127 @@
+# Test for DivField and GradField kernels and VectorDivPenaltyDirichletBC bcs.
+# This test uses Raviart-Thomas elements to solve a model div-grad problem
+# in H(div). The problem is simply a div-grad formulation, u = -grad(p), and
+# div(u) = f, of the standard Poisson problem div(grad(p)) = -f.
+# Manufactured solution: p = cos(k*x)*sin(k*y)*cos(k*z).
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 3
+    nx = 6
+    ny = 6
+    nz = 6
+    xmax =  1
+    ymax =  1
+    zmax =  1
+    xmin = -1
+    ymin = -1
+    zmin = -1
+    elem_type = HEX27
+  []
+[]
+
+[Variables]
+  [u]
+    family = RAVIART_THOMAS
+    order = FIRST
+  []
+  [p]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+  [lambda]
+    family = SCALAR
+    order = FIRST
+  []
+[]
+
+[Kernels]
+  [coefficient]
+    type = VectorFunctionReaction
+    variable = u
+    sign = negative
+  []
+  [gradient]
+    type = GradField
+    variable = u
+    p = p
```

Comment:
The convention of `p` and `u` as input parameter names in the `DivField` and `GradField` objects make sense in the context of this test formulation, but I am concerned that this might be confusing in a more general sense to a user instead of something like `coupled_variable` for them. @lindsayad do you have thoughts here?

Reply:
I think something like `coupled_scalar_field` and `coupled_vector_field` or something like that would be good. I don't personally like the `u` parameter for the vector field as that is used exclusively to denote the *scalar* field in Cockburn's mixed formulation literature. Also `u` is often regarded as the `variable` the kernel is acting on in MOOSE documentation

Reply:
Done.